### PR TITLE
add measurementDenominator to the key of statsVars

### PR DESCRIPTION
**Pull request by qlj-lijuan**

### CODE REVIEW

File: static/data/hierarchy_statsvar.json [hunk 1]
```diff
@@ -3,22 +3,22 @@
   "sv": "top",
   "l": "Demographics",
   "t": "p",
-  "c": 1784,
+  "c": 1785,
   "cd": [
    {
-    "sv": "Count_Person",
+    "sv": "dc/cv35g4gzkth83",
```
Comment (shifucun) [inline static/data/hierarchy_statsvar.json:9]: can you get all the stats var equivalent to Count_Person and prefer the human stats var, same for the rest of the list

File: static/data/hierarchy_statsvar.json [hunk 2]
```diff
@@ -6183,43 +6183,6 @@
            }
           ]
          },
-         {
```
Comment (shifucun) [inline static/data/hierarchy_statsvar.json:6186]: the order is changed, is this expected?

File: tools/pv_tree_generator/util.py [hunk 3]
```diff
@@ -171,6 +171,10 @@ def _read_stat_var():
                 raise Exception('constraint property:{} not found in statistical'
                                 'variable with dcid: {}'.format(property, dcid))
             prop_val[property] = sv_dict[property]
+        if "measurementDenominator" in sv_dict:
+            prop_val["md"] = sv_dict["measurementDenominator"]
+        if (dcid == "Count_Person_PerArea"):
```
Comment (shifucun) [inline tools/pv_tree_generator/util.py:176]: remove the logging?

File: tools/pv_tree_generator/build_tree_test.py [hunk 4]
```diff
@@ -27,7 +27,8 @@ class BuildTreeTest(unittest.TestCase):
     @staticmethod
     def get_sv():
         """pick a subset of stat_var dcids"""
-        level0 = ['MarriedPopulation', 'DivorcedPopulation', 'MalePopulation']
+        level0 = ['MarriedPopulation', 'DivorcedPopulation',
```
Comment (shifucun) [inline tools/pv_tree_generator/build_tree_test.py:30]: for future PR, can update test to use the new stats var as well

File: tools/pv_tree_generator/util.py [hunk 5]
```diff
@@ -134,6 +134,32 @@ def _read_pop_obs_spec():
     return result
 
 
+def removeDuplicateStatsVar(stat_vars):
```
Comment (shifucun) [inline tools/pv_tree_generator/util.py:137]: please  add some comments on the logic and rational?

File: tools/pv_tree_generator/util.py [hunk 6]
```diff
@@ -134,6 +134,32 @@ def _read_pop_obs_spec():
...
+    for key in stat_vars:
+        stat_vars_dup = collections.defaultdict(list)
+        for sv in stat_vars[key]:
+            full_key = [sv.pop_type, sv.mprop, sv.stats]
+            for prop, val in sv.pv.items():
+                full_key.append(prop)
```
Comment (shifucun) [inline tools/pv_tree_generator/util.py:143]: full_key.extend([prop, val])

File: tools/pv_tree_generator/util.py (hunk 3 above)
Comment (qlj-lijuan) [reply tools/pv_tree_generator/util.py]: removed

File: static/data/hierarchy_statsvar.json (hunk 1 above)
Comment (qlj-lijuan) [reply static/data/hierarchy_statsvar.json:9]: Added a function to remove duplicates

File: tools/pv_tree_generator/util.py (hunk 6 above)
Comment (qlj-lijuan) [reply tools/pv_tree_generator/util.py]: done.

File: tools/pv_tree_generator/util.py (hunk 5 above)
Comment (qlj-lijuan) [reply tools/pv_tree_generator/util.py:137]: Added

File: tools/pv_tree_generator/build_tree_test.py (hunk 4 above)
Comment (qlj-lijuan) [reply tools/pv_tree_generator/build_tree_test.py:30]: will do

**Pull request merged**